pkg/operator/ceph/object/spec.go

```diff
@@ -69,21 +68,29 @@ func (c *clusterConfig) createDeployment(rgwConfig *rgwConfig) (*apps.Deployment
 }
 
 func (c *clusterConfig) makeRGWPodSpec(rgwConfig *rgwConfig) (v1.PodTemplateSpec, error) {
+	shareProcessNamespace := true
 	podSpec := v1.PodSpec{
 		InitContainers: []v1.Container{
 			c.makeChownInitContainer(rgwConfig),
 		},
-		Containers: []v1.Container{
-			c.makeDaemonContainer(rgwConfig),
-		},
+		Containers:    []v1.Container{c.makeDaemonContainer(rgwConfig)},
 		RestartPolicy: v1.RestartPolicyAlways,
 		Volumes: append(
 			controller.DaemonVolumes(c.DataPathMap, rgwConfig.ResourceName),
 			c.mimeTypesVolume(),
 		),
-		HostNetwork:       c.clusterSpec.Network.IsHost(),
-		PriorityClassName: c.store.Spec.Gateway.PriorityClassName,
+		HostNetwork:           c.clusterSpec.Network.IsHost(),
+		PriorityClassName:     c.store.Spec.Gateway.PriorityClassName,
+		ShareProcessNamespace: &shareProcessNamespace,
```

Comment:
Why set it here if it's also set below on line 90? The other daemons didn't do this.

Reply:
Good catch! It's a leftover.